manager.spec.yaml

```diff
@@ -80,7 +80,7 @@ paths:
   /cves/{cve_id}/affected_systems:
     get:
       summary: Affected systems for a given CVE.
-      description: Report of affected systems for a given CVE.
+      description: Report of affected systems for a given CVE. Rule presence and rule key parameters are deprecated and will get obsolete.
```

Comment:
Can you please add: `deprecated: true` to `rule_presence` and `rule_key` instead of this comment?